src/modules/SetListCard/components/SetListCardHeader/SetListCardHeader.tsx

```diff
@@ -20,12 +21,20 @@ export const SetListCardHeader: React.FC<
   return (
     <header className="mb-4 flex items-center gap-2">
       <div className="flex flex-col items-center gap-[2px] rounded bg-slate-100 p-2">
-        <span className="text-[8px]/[8px] uppercase">{month}</span>
-        <span className="text-base/4 font-medium">{day}</span>
+        <Text asElement="span" fontSize="[8px]/[8px]" className="uppercase">
```

Comment:
Nitpick: this element isn't using a text style. Should it use an existing style or do I need to create a new style to match this and make it reusable? 🤔 (Same applies to lines `27` and `32`)

Reply:
I've added `FIXME` comment so it can be fixed in a future PR